server.py

```diff
@@ -169,7 +170,14 @@ def do_get_responses():
         responses.append(document)
 
     results['results'] = responses
-    return jsonify(results)
+    return Response(json.dumps(results, default=json_serial))
```

Comment:
I think it would be good to also pass `indent=2` to `json.dumps`- it's hard to read the json output otherwise